doc/architecture.rst

```diff
@@ -318,9 +318,10 @@ and uses it to sign requests to OSDs and to metadata servers in the cluster.
 
 
 The ``cephx`` protocol authenticates ongoing communications between the client
-machine and the Ceph servers. Each message sent between a client and server,
-subsequent to the initial authentication, is signed using a ticket that the
-monitors, OSDs and metadata servers can verify with their shared secret.
+machine and the Ceph servers. After initial authentication, each message sent
```

Comment:
@anthonyeleven: What do you think of this word, "servers"? It seems to me that we often say "hosts" or "nodes" in our docs, and suddenly here we are busting out a third synonym just to confuse our readers. What do you think?

Reply:
Why stop confusing our readers now? ;). 

I suggest 
>between clients and Ceph daemons.

Reply:
This is exactly what I was hoping that you'd suggest, in exactly the words I was hoping that you'd suggest it.

Reply:
:p

Reply:
One might then wonder why you didn't just go with that in the first place ;)

Reply:
I wanted to keep your expertise free of the taint of any suggestion from me, so that it was pure when it was tapped.